grails-app/views/finder/_auv_functions_js.gsp

```diff
@@ -311,13 +309,13 @@
                 + "&version=1.0.0&maxfeatures=1&sortby=" + variable + "+a";
 
 
-        var xmlDoc = getXML(url);
+        getXML(url, function(doc) {
+            var xmlDoc = doc;
```

Comment:
Ah yes, I meant to write a comment about this, as I saw this too.  In short, "probably not" - but I didn't want to go changing too much.  There are some interesting scoping issues going on in this javascript.